Bottom: 1e2a7fee24f00858385f8556f6f6a4c2b91c20dc
Top:    073207cf2269dd797942b6393b4e95afcd3fc741
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-12-25 11:56:14 +0800

Refresh of tun-tap-use-ptr_ring-instead

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 67f6ebc..c7fdfe5 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -801,7 +801,7 @@ static void handle_rx(struct vhost_net *net)
 			 * they refilled. */
 			goto out;
 		}
-		if (nvq->rx_array)
+		if (nvq->rx_ring)
 			msg.msg_control = vhost_net_buf_consume(&nvq->rxq);
 		/* On overrun, truncate and discard */
 		if (unlikely(headcount > UIO_MAXIOV)) {
